### DISCUSSION TITLE
Does deleting an email address delete your repo?

### DISCUSSION THREAD

**Discussion by leonardmoses** [community/community#35817] [Repositories] [answered]
If I created a repository on my gitHub account. Then later added another email address as a new primary email address and deleted the original email address. Will that repository created from the original email address no longer belong to my gitHub account? If so how can I clean up older inactive email addresses?

<img width="449" alt="Screen Shot 2022-10-11 at 3 07 17 PM" src="https://user-images.githubusercontent.com/78382082/195177875-c678ce09-7374-48ef-b9a7-2fcf03094657.png">

**Comment by airtower-luna** [comment, marked as answer]
> Will that repository created from the original email address no longer belong to my gitHub account?

The *repository* will still be yours, but *commits* with your old address as author or committer email won't be counted as yours any more.

> If so how can I clean up older inactive email addresses?

I believe the recommended approach is "don't". :sweat_smile: You can configure which of your addresses should get notifications etc., so you can just leave the old addresses on your account.

**Reply by leonardmoses** [reply]
Ahh... seems like I have to leave it be then. RIP my OCD. Thanks @airtower-luna !

**Reply by airtower-luna** [reply]
Glad I could help! Marking the answer would be appreciated. :slightly_smiling_face:

**Reply by leonardmoses** [reply]
Got it thank you!

**Comment by Yvonnelim99** [comment]
How to remove a email address,will you teach me?

**Reply by airtower-luna** [reply]
Got to your account settings, and select "Emails". There should be a list of all the addresses on your account, and a red trash can icon to the right of each.

Mind the issues discussed above though, if you have used the address as a commit author email it'll usually be better to just pick another primary/backup address and leave the old address on your account.

**Reply by Yvonnelim99** [reply]
Thx you sir, I'm sure I'm going to take care my GitHub's account.

**Reply by airtower-luna** [reply]
You're welcome! No sir here though… :wink: :woman_technologist:

**Reply by Yvonnelim99** [reply]
> You're welcome! No sir here though… 😉 👩‍💻

Oh.....Ok np.
